src/sentry/tagstore/snuba/backend.py

```diff
@@ -920,12 +920,18 @@ def get_tag_value_paginator_for_projects(
     ):
         from sentry.api.paginator import SequencePaginator
 
-        if not order_by == "-last_seen":
+        if not (order_by == "-last_seen" or order_by == "-count"):
             raise ValueError("Unsupported order_by: %s" % order_by)
 
+        # We need to replace `-count` into `-times_seen`, because
+        # internally we can not order by `count` we can only by `times_seen`.
+        if order_by == "-count":
+            order_by = "-times_seen"
```

Comment:
If we cannot order by `count`, why do we need to add it to the API?

Reply:
yeah it's a good question, but when i proposed to add order by count(times_seen) - it confuse users of API. So i decided to keep API simple, and internally replace to correct column.
```
GET 200 /api/0/organizations/sentry/tags/environment/values/?includeTransactions=1&project=1&sort=-count 
```

transformed into:
```
  SELECT (environment AS _snuba_environment), (count() AS _snuba_times_seen), (min((timestamp AS _snuba_timestamp)) AS _snuba_first_seen), (max(_snuba_timestamp) AS _snuba_last_seen)
   FROM discover_local SAMPLE 1000000.0 PREWHERE notEquals(_snuba_environment, '')
  WHERE equals(deleted, 0)
    AND greaterOrEquals(_snuba_timestamp, toDateTime('2022-04-23T11:46:21', 'Universal'))
    AND less(_snuba_timestamp, toDateTime('2022-07-22T11:46:21', 'Universal'))
    AND in((project_id AS _snuba_project_id), tuple(1))
    AND in(_snuba_project_id, tuple(1))
  GROUP BY _snuba_environment
  ORDER BY _snuba_times_seen DESC
  LIMIT 1000
 OFFSET 0
```

Reply:
Right. I was confusing `last_seen` with `times_seen` and thought for a moment that we always order by the same field, but this makes sense.